base/applications/kbswitch/kbswitch.c

```diff
@@ -92,15 +92,17 @@ GetSystemLibraryPath(LPTSTR szPath, SIZE_T cchPath, LPCTSTR FileName)
     return TRUE;
 }
 
+/* FIXME: Use <shlwapi_undoc.h> and delete this */
+HRESULT WINAPI SHLoadRegUIStringW(HKEY hkey, LPCWSTR value, LPWSTR buf, DWORD size);
+
 static BOOL
 GetLayoutName(LPCTSTR szLayoutNum, LPTSTR szName, SIZE_T NameLength)
 {
     HKEY hKey;
     DWORD dwBufLen;
-    TCHAR szBuf[MAX_PATH], szDispName[MAX_PATH], szIndex[MAX_PATH], szPath[MAX_PATH];
+    TCHAR szBuf[MAX_PATH], szDispName[MAX_PATH];
```

Comment:
```suggestion
    TCHAR szBuf[MAX_PATH];
    WCHAR szDispName[MAX_PATH];
```
szDispName is used with explicilty unicode SHLoadRegUIStringW.